packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -138,52 +83,12 @@ - (void)windowScene:(UIWindowScene *)windowScene
   [[NSNotificationCenter defaultCenter] postNotificationName:RCTWindowFrameDidChangeNotification object:self];
 }
 
-- (RCTColorSpace)defaultColorSpace
-{
-  return RCTColorSpaceSRGB;
-}
-
-#pragma mark - New Arch Enabled settings
-
-- (BOOL)newArchEnabled
-{
-#if RCT_NEW_ARCH_ENABLED
-  return YES;
-#else
-  return NO;
-#endif
-}
-
-- (BOOL)turboModuleEnabled
-{
-  return [self newArchEnabled];
-}
-
-- (BOOL)fabricEnabled
-{
-  return [self newArchEnabled];
-}
-
-- (BOOL)bridgelessEnabled
-{
-  return [self newArchEnabled];
-}
-
-- (NSURL *)bundleURL
-{
-  [NSException raise:@"RCTAppDelegate::bundleURL not implemented"
-              format:@"Subclasses must implement a valid getBundleURL method"];
-  return nullptr;
-}
-
-#pragma mark - RCTHostDelegate
+#pragma mark - Bridge and Bridge Adapter properties
```

Comment:
we can probably remove this comment, right?

Reply:
Yes!